src/components/circle.vue

```diff
@@ -0,0 +1,108 @@
+/* vim: set softtabstop=2 shiftwidth=2 expandtab : */
+
+<script>
+
+import _ from 'lodash';
+
+import eventBinder from '../utils/eventsBinder.js'
+import propsBinder from '../utils/propsBinder.js'
+
+const props = {
+  clickable: {
+    type: Boolean,
+  },
+  draggable: {
+    type: Boolean
+  },
+  editable: {
+    type: Boolean
+  },
+  center: {
+    type: Object,
+    twoWay: true,
+    required: true
+  },
+  fillColor: {
```

Comment:
This certainly make sense but according to the documentation this property does not exists in the google documentation. `fillColor` and other properties are indeed part of of the of the `CircleOption`. The aim of this library is to stick as close as possible to the the google maps API. 

In order to keep the repo consistent, and avoid any divergence with the Google API. Id rather have an option object instead of multiple properties like (`fillColor` and `fillOpacity`).

This strategy has also other advantages: 

- You can write code with `vue-google-maps` just by reading the google documentation.
- If google add a new property to `CircleOption` There is no need to update `vue-google-maps`
- There is less code to maintain on our side

If you disagree with this and think this policy must change we can discuss it in a separated issue